integration-tests/api/__tests__/admin/product-category.ts

```diff
@@ -234,6 +234,71 @@ describe("/admin/product-categories", () => {
     })
   })
 
+  describe("POST /admin/product-categories", () => {
+    beforeEach(async () => {
+      await adminSeeder(dbConnection)
+    })
+
+    afterEach(async () => {
+      const db = useDb()
+      return await db.teardown()
+    })
+
+    it("throws an error if required fields are missing", async () => {
+      const api = useApi()
+
+      const error = await api.post(
+        `/admin/product-categories`,
+        {},
+        adminHeaders
+      ).catch(e => e)
+
+      expect(error.response.status).toEqual(400)
+      expect(error.response.data.type).toEqual("invalid_data")
+      expect(error.response.data.message).toEqual(
+        "name should not be empty, name must be a string"
+      )
+    })
+
+    it("successfully creates a product category", async () => {
+      productCategoryParent = await simpleProductCategoryFactory(dbConnection, {
+        name: "category parent",
+        handle: "category-parent",
+      })
+
+      const api = useApi()
+
+      const response = await api.post(
```

Comment:
non blocking: should we add a tests without passing the handle to also assert that this behaviour do not broke at some point?